project.clj

```diff
@@ -23,15 +23,15 @@
                  [metosin/compojure-api "1.1.14"]
                  [org.cyverse/async-tasks-client "0.0.5"]
                  [org.cyverse/authy "3.0.1"]
-                 [org.cyverse/clj-kondo-exports "0.1.0-SNAPSHOT"]
+                 [org.cyverse/clj-kondo-exports "0.1.0"]
```

Comment:
Not sure why I failed to push this commit to the main fork before, but I'm glad that this was caught before next week's maintenance.